roles/keystone/templates/etc/keystone/policy.json

```diff
@@ -178,14 +171,14 @@
     "identity:delete_protocol": "rule:admin_required",
 
     "identity:create_mapping": "rule:admin_required",
-    "identity:get_mapping": "rule:admin_required",
-    "identity:list_mappings": "rule:admin_required",
+    "identity:get_mapping": "rule:admin_or_cloudadmin",
+    "identity:list_mappings": "rule:admin_or_cloudadmin",
     "identity:delete_mapping": "rule:admin_required",
-    "identity:update_mapping": "rule:admin_required",
+    "identity:update_mapping": "rule:admin_or_cloudadmin",
```

Comment:
Would opening up mappings to cloud_admin somehow allow them to create a mapping to the `admin` role?